web3c/contract_factory.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @returns a constructor to create a web3.eth.Contract object. The `Contract` object
```

Comment:
@returns goes with @param's after an initial description of the function.

Reply:
What extra information would be conveyed with such a description?